x-team/xp-ui
# Include missing dependency TextareaEditor

**Release Type:** *non-breaking feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-3051

## Description

Include missing dependency TextareaEditor

## Checklist

**Before submitting a pull request,** please make sure the following is done:

- [x] fill out `Is a breaking change` and `Release Type Reason` (if required) fields for the ticket. This is accessible when editing an issue in JIRA

## Related PRs



branch | PR
------ | ------
xp_branch_name | [XP](https://github.com/x-team/xp/pull/X)
xp_api_branch_name | [XP API](https://github.com/x-team/xp-api/pull/X)

## Steps to Test or Reproduce

- Make sure to have the latest node_modules `rm -r node_modules && npm install`
```
✔ ~/Sites/x-team/xp-registration/site [gh-issue-43-implement-shouldSubmitPortfolio] 
$ cp ../../xp-ui/lib/xp-ui.js node_modules/xp-ui/lib/xp-ui.js
```

I should solve the bug found at https://github.com/x-team/xp-registration/pull/44#pullrequestreview-239082357

## Impacted Areas in Application

XP-Registration app
